sailthru/sailthru_content/services/sailthru_translation_service.py

```diff
@@ -69,7 +71,7 @@ def _create_course_vars(self, course, course_run, url, site_name, program_dictio
                 # multiple seat types the one with the higher precedence will be added
                 # in course_type
                 if not sailthru_content_vars.get('course_type', None) or \
-                        seat_priority.index(sailthru_content_vars['course_type']) < seat_priority.index(seat['type']):
+                        seat_priority.index(sailthru_content_vars['course_type']) > seat_priority.index(seat['type']):
```

Comment:
yes it is a bug after testing it seems like audit mode was having higher priority as compared to verified so I made this change.